api/lib/api/clients/language_service.ex

```diff
@@ -37,8 +37,20 @@ defmodule Api.Clients.LanguageService do
     end
   end
 
-  def get_vocab(language, word) do
-    url = "/v1/vocabulary/" <> serialize_language(language) <> "/" <> word
-    get(url)
+  def definition(language, word) do
+    url = "/v1/definition/" <> serialize_language(language) <> "/" <> word
+    case get(url) do
+      {:ok, %{body: body}} -> {:ok, resolve_definition_in_english(body)}
+      _ -> :error
+    end
+  end
+
+  # We will eventually resolve definitions in more than one base language
+  defp resolve_definition_in_english(response_body) do
+    response_body
+    |> Enum.filter( fn word -> Map.has_key?(word, "definitions") end)
+    |> Enum.flat_map( fn word -> Map.fetch!(word, "definitions") end)
+    |> Enum.filter( fn definition -> Map.has_key?(definition, "text") end)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There is no whitespace around parentheses/brackets most of the time, but here there is.](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5104474)